src/JuMP.jl

```diff
@@ -32,8 +32,8 @@ export
     affToStr, quadToStr, conToStr, chgConstrRHS,
 
 # Macros and support functions
-    @addConstraint, @addConstraints, @defVar,
-    @defConstrRef, @setObjective, addToExpression, @defExpr,
+    @addConstraint, @defConstraint, @addConstraints, @defConstraints,
```

Comment:
My codes looks like
```jl
function foo()
    return [
        dˣ[i,j] ≤ cˣ[i] - cˣ[j] + 2*(Lˣ - lbˣ[i] - lbˣ[j])*(1 - zˣ[j,i])
        dˣ[i,j] ≤ cˣ[j] - cˣ[i] + 2*(Lˣ - lbˣ[i] - lbˣ[j])*(1 - zˣ[i,j])
        dʸ[i,j] ≤ cʸ[i] - cʸ[j] + 2*(Lʸ - lbʸ[i] - lbʸ[j])*(1 - zʸ[j,i])
        dʸ[i,j] ≤ cʸ[j] - cʸ[i] + 2*(Lʸ - lbʸ[i] - lbʸ[j])*(1 - zʸ[i,j])
    ]
end
```
so I was planning on making something like
```jl
function foo()
    return @defConstraints(begin
        dˣ[i,j] ≤ cˣ[i] - cˣ[j] + 2*(Lˣ - lbˣ[i] - lbˣ[j])*(1 - zˣ[j,i])
        dˣ[i,j] ≤ cˣ[j] - cˣ[i] + 2*(Lˣ - lbˣ[i] - lbˣ[j])*(1 - zˣ[i,j])
        dʸ[i,j] ≤ cʸ[i] - cʸ[j] + 2*(Lʸ - lbʸ[i] - lbʸ[j])*(1 - zʸ[j,i])
        dʸ[i,j] ≤ cʸ[j] - cʸ[i] + 2*(Lʸ - lbʸ[i] - lbʸ[j])*(1 - zʸ[i,j])
    end)
end
```
(though it currently wouldn't). Open to other suggestions, though.